lib/stm32wb0/README

```diff
@@ -0,0 +1,123 @@
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dtm_cmd_stack_en.h
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/crash_handler.h
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/ble_conf.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_gatt.h
+      Middlewares/ST/STM32_BLE/stack/include/uuid.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_stack.h
+      Middlewares/ST/STM32_BLE/ble.h
+      Middlewares/ST/STM32_BLE/stack/include/bleplat_cntr.h
+      Middlewares/ST/STM32_BLE/stack/include/bleplat.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_const.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_status.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_stack_user_cfg.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_api.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_events.h
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dm_alloc.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dtm_cmd_db.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/aci_adv_nwk.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/Target/bleplat_cntr.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/Target/bleplat.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/PKAMGR/Src/pka_manager.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Interfaces/hw_aes.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Interfaces/hw_pka.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Interfaces/hw_rng.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/blue_unit_conversion.s
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/osal_memcpy.s
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/adv_buff_alloc_tiny.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/transport_layer.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dtm_preprocess_events.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/adv_buff_alloc.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/RADIO_utils/Src/RADIO_utils.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/miscutil.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/pawr_buff_alloc.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dtm_cmds.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/Core/Src/stm32wb0x_hal_msp.c
+      Middlewares/ST/STM32_BLE/stack/config/ble_stack_user_cfg.c
```

Comment:
I see more than those files in the commit (Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/Core  files for instance).
Can you verify ?

Also it would be easier if the list was sorted.

Reply:
Actually, it was @msmttchr's idea to deliver the library with as less changes as possible so as to make it easier for future deliveries.

Ok, I'll sort them.

Reply:
> Actually, it was @msmttchr's idea to deliver the library with as less changes as possible so as to make it easier for future deliveries.

Minimal changes, I fully agree.
But, till now, for WB and WBA we've only pushed the minimal set of required files to ease the update and review process. Note that using the script this isn't too much of a hassle.

Reply:
Done.